src/views/Community/Managers.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+import {
+    Avatar,
+    Box,
+    Button,
+    Card,
+    Divider,
+    Grid,
+    Icon,
+    Tab,
+    TabList,
+    Tabs,
+    Text
+} from '@impact-market/ui';
+
+
+const Managers = ({ community } : any) => {
+    
+    // TEMPORARY OBJECT
+    const managers = [
+        {
+            avatar:'https://picsum.photos/120',
+            beneficiaries:'Onboarded 22 Beneficiaires.',
+            id:'0x43D2...a12f7',
+            mail:'johndoe@myemail.com',
+            manager:'Manager from May 12, 2020 – Present',
+            name:'Paulo Silva Queirós',
+            phone:'+1 669 2319892'   
+        },
+        {
+            avatar:'https://picsum.photos/120',
+            beneficiaries:'Onboarded 22 Beneficiaires.',
+            id:'0x43D2...a12f7',
+            mail:'johndoe@myemail.com',
+            manager:'Manager from May 12, 2020 – Present',
+            name:'Paulo Silva Queirós',
+            phone:'+1 669 2319892'   
+        },
+        {
+            avatar:'https://picsum.photos/120',
+            beneficiaries:'Onboarded 22 Beneficiaires.',
+            id:'0x43D2...a12f7',
+            mail:'johndoe@myemail.com',
+            manager:'Manager from May 12, 2020 – Present',
+            name:'Paulo Silva Queirós',
+            phone:'+1 669 2319892'   
+        }
+    ]
```

Comment:
If you want to approve this and want to add mock data, create a separate file where we return a promise wrapped with a timeout and create the request as we should be fetching data from API.